### DISCUSSION TITLE
size of embedded drawings in md documents?

### DISCUSSION THREAD

**Discussion by Stefan-59** [zsviczian/obsidian-excalidraw-plugin#678] [General]
To me it seems, as if any drawing is resized (zoom+ or zoom-) so it fit's the size set in the settings under 'Default width of embedded (transcluded) image.

In your (wonderful!) video about creating the 'book on a page' there were some smaller images: 
![image](https://user-images.githubusercontent.com/61938696/175909401-2a485f7c-2b33-46fe-a7fe-b9748ea28bad.png)
Are these embedded PNG or also Excalidraw?
If the latter, how did you manage to display them smaller than the other images?

Stefan

**Comment by Stefan-59** [comment]
I just realized, that the height of the embedded drawing seems to be dependent of the width and height of the image.
![image](https://user-images.githubusercontent.com/61938696/175917297-57df9c5c-81c4-421b-ab3c-b73f04bb7016.png)
Is it possible to avoid that the single (small) icon is zoomed that much?
Can I zoom down large images but not zoom up small images?

**Reply by Stefan-59** [reply]
Yes, it is possible, I found out myseld (RTFM 😜)
I simply add the desired size (i.e. '|150') and that's it. Too simple...

**Comment by mwtindol** [comment]
I'm fairly new to Excalidraw but in my short time of using it I'm primary embedding files that also contain embedded images I want to display. I have used the '|width x length' method to resize the embed in the drawing but my image scaling and text scaling seem to always be odds with one another. I literally spent hours chasing my tail trying to get things just right. Finally, I stumbled upon resizing the image directly in the file I was embedding in my Excalidraw drawing. That provided a bit more control over the scaling in Excalidraw. 
What you're seeing below is a embedded image within an embed file within a drawing. Note how the text and image in the drawing are still a bit stretched. At some point I decided this was "good enough". Hope this helps others.
![image](https://user-images.githubusercontent.com/49814575/195929766-2d57ba6a-2de5-4537-9df8-642d05875cad.png)
